docs/pages/operations-guide.md

```diff
@@ -320,7 +320,7 @@ name: secret-app/instance1
 pods:
```

Comment:
If your application is designed to read private information from an environment variable (most are designed to read private information from a file), you may prefer environment-variable based secret. Alternatively, you can use file-based secret, and read from the secret file and put to an environment variable using an initialization script.  File-based and environment-based secrets are doing the same thing. 

Mesos protobuf definition lets you define both environment and file based secrets even for the same secret, so our SDK yaml will also let you do that.

"The SDK allows secrets to be exposed to pods as a file and/or as an environment variable."